app/collections/Classrooms.coffee

```diff
@@ -0,0 +1,17 @@
+Classroom = require 'models/Classroom'
+CocoCollection = require 'collections/CocoCollection'
+
+module.exports = class Classrooms extends CocoCollection
+  model: Classroom
+  url: '/db/classroom'
+  
+  initialize: ->
+    @on 'sync', ->
+      for classroom in @models
+        classroom.capitalizeLanguageName()
```

Comment:
Huh. Yeah, it's not in the docs, but by default the context is the emitter.

https://github.com/jashkenas/backbone/blob/c4056792ed19fef295e73ca42643d9b149ccf391/backbone.js#L210